utilities/test_suite/HOST/runAudioTests.py

```diff
@@ -0,0 +1,294 @@
+# Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import os
+import subprocess  # nosec
+import argparse
+import sys
+import datetime
+import shutil
+
+# Set the timestamp
+timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+
+cwd = os.getcwd()
+inFilePath = os.path.join(os.path.dirname(cwd), 'TEST_AUDIO_FILES', 'eight_samples_single_channel_src1')
+
+# Checks if the folder path is empty, or is it a root folder, or if it exists, and remove its contents
+def validate_and_remove_files(path):
+    if not path:  # check if a string is empty
+        print("Folder path is empty.")
+        exit()
+
+    elif path == "/*":  # check if the root directory is passed to the function
+        print("Root folder cannot be deleted.")
+        exit()
+
+    elif os.path.exists(path):  # check if the folder exists
+        # Get a list of files and directories within the specified path
+        items = os.listdir(path)
+
+        if items:
+            # The directory is not empty, delete its contents
+            for item in items:
+                item_path = os.path.join(path, item)
+                if os.path.isfile(item_path):
+                    os.remove(item_path)
+                elif os.path.isdir(item_path):
+                    shutil.rmtree(item_path)     # Delete the directory if it exists
+
+    else:
+        print("Path is invalid or does not exist.")
+        exit()
+
+# Check if the folder is the root folder or exists, and remove the specified subfolders
+def validate_and_remove_folders(path, folder):
```

Comment:
For python also we have some common functions which can be used across test suites
May be we can have this common code in a file called commons.py and use it across?

Not necessarily now, but better we do it this way
@r-abishek kindly request you to share your thoughts on this

Reply:
Agreed, keep similar names for the cpp and python common headers. Use the word test_suite or say testSuiteCommon.py
Please add this one to your running test suite list of tasks.